_posts/2021-07-01-wildfly-preview-bootable-jar.adoc

```diff
@@ -0,0 +1,207 @@
+---
+layout: post
+title: Run WildFly Preview of Jakarta EE 9.1 with Bootable Jar
+date: 2021-07-01
+tags: wildfly jakarta-ee9 bootable-jar cloud
+synopsis: This article demonstrates how to run a Jakarta EE 8 application on WildFly Preview for Jakarta EE 9.1 using the Bootable Jar.
+author: jmesnil
+---
+
+https://jakarta.ee/[Jakarta EE 9.1] has been recently released and WildFly 24.0.0.Final provides a preview to run existing Jakarta EE 8 applications on it.
+
+In this blog, we will take a simple existing Jakarta EE 8 applications (the WildFly https://github.com/wildfly/quickstart/tree/master/helloworld-rs[JAX-RS quickstart]) and modify the project structure to run it on WildFly with the EE 9.1 Preview.
+
+== Compiling with Jakarta EE 8, Running with Jakarta EE 9.1
+
+We will not change the code of the application itself that is compiled against Jakarta EE 8. We will only change its execution to run on top of Jakarta EE 9.1.
+
+The application in itself is a simple JAX-RS application that has a `/rest/json` HTTP endpoint.
+The application code imports the `javax.ws.rs.ApplicationPath` annotation. The `javax.ws.rs` namespace means that the code is compiled against Jakarta EE 8. In Jakarta EE 9.1, the namespace for JAX-RS became `jakarta.ws.rs`
```

Comment:
You could mention Jakarta RESTful Web Services, perhaps with a link like:

the namespace for JAX-RS became ([Jakarta RESTful Web Services](https://jakarta.ee/specifications/restful-ws/3.0)) `jakarta.ws.rs`

Reply:
fixed, thanks